src/main/java/MainWindow.java

```diff
@@ -6,48 +6,38 @@
+        String greeting =
+                "THIS IS DA                                                  \r\n                                                         ttt:::t                                                  \r\n                                                         t:::::t                                                  \r\n                                                         t:::::t                                                  \r\n    ssssssssss   uuuuuu    uuuuuunnnn  nnnnnnnn    ttttttt:::::ttttttt        eeeeeeeeeeee        cccccccccccccccc\r\n  ss::::::::::s  u::::u    u::::un:::nn::::::::nn  t:::::::::::::::::t      ee::::::::::::ee    cc:::::::::::::::c\r\nss:::::::::::::s u::::u    u::::un::::::::::::::nn t:::::::::::::::::t     e::::::eeeee:::::ee c:::::::::::::::::c\r\ns::::::ssss:::::su::::u    u::::unn:::::::::::::::ntttttt:::::::tttttt    e::::::e     e:::::ec:::::::cccccc:::::c\r\n s:::::s  ssssss u::::u    u::::u  n:::::nnnn:::::n      t:::::t          e:::::::eeeee::::::ec::::::c     ccccccc\r\n   s::::::s      u::::u    u::::u  n::::n    n::::n      t:::::t          e:::::::::::::::::e c:::::c             \r\n      s::::::s   u::::u    u::::u  n::::n    n::::n      t:::::t          e::::::eeeeeeeeeee  c:::::c             \r\nssssss   s:::::s u:::::uuuu:::::u  n::::n    n::::n      t:::::t    tttttte:::::::e           c::::::c     ccccccc\r\ns:::::ssss::::::su:::::::::::::::uun::::n    n::::n      t::::::tttt:::::te::::::::e          c:::::::cccccc:::::c\r\ns::::::::::::::s  u:::::::::::::::un::::n    n::::n      tt::::::::::::::t e::::::::eeeeeeee   c:::::::::::::::::c\r\n s:::::::::::ss    uu::::::::uu:::un::::n    n::::n        tt:::::::::::tt  ee:::::::::::::e    cc:::::::::::::::c\r\n  sssssssssss        uuuuuuuu  uuuunnnnnn    nnnnnn          ttttttttttt      eeeeeeeeeeeeee      cccccccccccccccc";
```

Comment:
Maybe you want to add a comment on this part? I got a little confuse and understood what it was supposed to do after running the program.